### DISCUSSION TITLE
Changelog for 5.0.0 we will soon need to enable developer mode

### DISCUSSION THREAD

**Discussion by royemosby** [Tampermonkey/tampermonkey#1947] [Q&A]
>It will soon be necessary to enable developer mode to run userscripts via Tampermonkey.
Instructions on how to enable it can be found [here](https://developer.chrome.com/docs/extensions/reference/api/userScripts#developer_mode_for_extension_users).

Is there a reference that provides additional context? Browser versions/timelines/why/etc...

**Comment by derjanb** [comment]
Sure. https://groups.google.com/a/chromium.org/g/chromium-extensions/c/bdnjlMNzQqA/m/h18kuLBbAAAJ
